package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "postinstall": "patch-package --patch-dir ./other/patches",
     "build": "run-s build:*",
-    "build:icons": "rmx svg-sprite other/svg-icons app/components/ui/icon.tsx --template=./other/svg-icon-template.txt",
+    "build:icons": "tsx ./other/build-icons.ts",
```

Comment:
I wasn't quite sure where to put this file. In the svg-icons folder made sense, in the Sly folder kinda made sense, but this spot matches the build:server command so I copied the filename format of that too